demo/src/app.js

```diff
@@ -263,6 +264,74 @@ const validateUser = (user) => {
     );
 };
 
+const TableTabs = () => {
```

Comment:
Maybe put this component to another file ?
Then rename it too, because it looks more as a Table, there is no Tabs in here.